src/Stage.js

```diff
@@ -53,33 +53,94 @@ function Stage(soundscape, priorActionQueueList, onEnd) {
 
     this.world.SetContactListener(listener);
 
-    var walls = [
-        [
-            { x: -1, y: 20 },
-            { x: 25, y: 18 },
-            { x: 50, y: 17 },
-            { x: 75, y: 18 },
-            { x: 101, y: 20 },
-            { x: 101, y: 101 },
-            { x: -1, y: 101 }
-        ], [
-            { x: -1, y: -20 },
-            { x: -1, y: -101 },
-            { x: 101, y: -101 },
-            { x: 101, y: -20 },
-            { x: 75, y: -18 },
-            { x: 50, y: -17 },
-            { x: 25, y: -18 }
-        ], [
-            { x: 25, y: 10 },
-            { x: 25, y: -10 },
-            { x: 60, y: 10 }
-        ], [
-            { x: 75, y: 10 },
-            { x: 40, y: -10 },
-            { x: 75, y: -10 }
-        ]
-    ];
+    var fs = require('fs');
```

Comment:
This and L61 could be outside of this constructor - didn't we run into problems with doing the read in non-global scope before?